astrapy/db.py

```diff
@@ -113,6 +113,10 @@ def __init__(
         self.collection_name = collection_name
         self.base_path = f"{self.astra_db.base_path}/{self.collection_name}"
 
+        # Set read concern to "NEAREST"
```

Comment:
Is read concern the same as read preference? The mongo [docs](https://pymongo.readthedocs.io/en/stable/api/pymongo/collection.html#pymongo.collection.Collection.read_preference) seem to imply that they are not.
In that case the comment may seem confusing here